test/e2e/hive.go

```diff
@@ -44,6 +44,7 @@ var _ = Describe("Hive-managed ARO cluster", func() {
 	})
 
 	It("has been properly created/adopted by Hive", func(ctx context.Context) {
+		Skip("Hive test currently failed.")
```

Comment:
just checking - what would be the steps to enable this test in the future?

Reply:
I think leaving a TODO with a comment on the Jira that replaces could be helpful in this case.

Reply:
Here is the Jira : https://issues.redhat.com/browse/ARO-12231